<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmMainMenuGUI</class>
 <widget class="QDialog" name="frmMainMenuGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>easy fMRI (1.0)</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>371</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="text">
    <string>easy fMRI</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPreprocess">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Preprocessing (FSL)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFeatureAnalysis">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Feature Analysis</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnModelAnalysis">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Model Analysis</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnVisualization">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Visualization (AFNI)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnExit">
   <property name="geometry">
    <rect>
     <x>191</x>
     <y>340</y>
     <width>150</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAbout">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>150</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
